backend/database_handler/transactions_processor.py

```diff
@@ -155,4 +167,8 @@ def set_transaction_result(self, transaction_hash: str, consensus_data: dict):
         transaction.status = TransactionStatus.FINALIZED
         transaction.consensus_data = consensus_data
 
-        self.session.commit()
+        print(
```

Comment:
@AgustinRamiroDiaz wasn't this moved to the consensus? and why is there a bare print here?